corehq/apps/hqwebapp/forms.py

```diff
@@ -21,10 +23,13 @@
 
 LOCKOUT_MESSAGE = mark_safe(_('Sorry - you have attempted to login with an incorrect password too many times. Please <a href="/accounts/password_reset_email/">click here</a> to reset your password or contact the domain administrator.'))
 
+
 class EmailAuthenticationForm(NoAutocompleteMixin, AuthenticationForm):
     username = forms.EmailField(label=_("E-mail"), max_length=75,
                                 widget=forms.TextInput(attrs={'class': 'form-control'}))
     password = forms.CharField(label=_("Password"), widget=forms.PasswordInput(attrs={'class': 'form-control'}))
+    if settings.ENABLE_DRACONIAN_SECURITY_FEATURES:
```

Comment:
In case you would like to add this back, it should probably be skipped for mobile workers?

Reply:
Its actually added specifically for mobile workers. They are not locked out and was raised as a security concern.
And at this point we don't know if user will be a mobile user or web user, its a login page